tests/do.sh.in

```diff
@@ -18,6 +18,14 @@ if [ "${NDPI_FORCE_UPDATING_UTESTS_RESULTS}" = "1" ]; then
    FORCE_UPDATING_UTESTS_RESULTS=1
 fi
 
+#For paralell tests you need `parallel` from GNU, not from `moreutils` package!
```

Comment:
Maybe check if `parrallel` is from GNU via:
`if parallel -V | grep -qoE 'GNU parallel'; then echo 'GNU'; else echo 'not GNU'; fi`

Reply:
Useful! Thanks!